src/apps/checklist-button/checklist-material-button.slice.jsx

```diff
@@ -76,7 +102,7 @@ export const checklistMaterialButtonSlice = createSlice({
     },
     [removeFromListAction.fulfilled]: (state, action) => {
       state.status[action.meta.arg.materialId] = "finished";
-      state.onList[action.meta.arg.materialId] = "off";
+      state.onList[action.meta.arg.materialId] = "on";
```

Comment:
... basically a continuation of my comment above.

When an `removeFromListAction` is fulfilled I would expect the material to be `off` the list.